IHP/IDE/SchemaDesigner/View/Layout.hs

```diff
@@ -368,3 +443,6 @@ functionIcon = preEscapedToHtml [plain|<svg xmlns="http://www.w3.org/2000/svg" v
 
 -- | https://github.com/postgres/pgadmin4/blob/master/web/pgadmin/misc/static/explain/img/ex_unknown.svg
 unknownIcon = preEscapedToHtml [plain|<svg id="_1" data-name="1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><defs><style>.cls-unknown-1{fill:#1cafe4;}</style></defs><title>unknown</title><path class="cls-unknown-1" d="M23.34,13.2a14.76,14.76,0,0,1,8.16-2.14,17.57,17.57,0,0,1,10.56,3q4.2,3,4.2,9a10.18,10.18,0,0,1-1.82,6.16,21,21,0,0,1-4.09,3.88l-2,1.55a6,6,0,0,0-2.16,3A12.5,12.5,0,0,0,35.83,41H28.2a22.11,22.11,0,0,1,.9-6.57,11.57,11.57,0,0,1,3.76-4.18l2-1.6A7.17,7.17,0,0,0,36.53,27a5.67,5.67,0,0,0,1.12-3.4,6.58,6.58,0,0,0-1.25-3.9q-1.25-1.76-4.56-1.76t-4.62,2.17a8.34,8.34,0,0,0-1.36,4.5H17.74Q18.08,16.55,23.34,13.2ZM28,44.81h8.41v8.13H28Z"/></svg>|]
+
+-- | https://fonts.google.com/icons?icon.query=shield
```

Comment:
TIL about https://fonts.google.com/icons! Really cool.

Why `preEscapedToHtml` instead of `hsx` btw?

Reply:
[HSX has a built-in whitelist for tags and attribute names](https://github.com/digitallyinduced/ihp/blob/master/IHP/HSX/Parser.hs#L264) (to avoid hard to find spelling mistakes). The whitelist doesn't include most SVG tags and attributes and so `[hsx||]` would fail here for most icons, so we use `preEscapedToHtml` to avoid that.